### ISSUE TITLE
Broken metarefresh in 1.19.2 and 1.19.3 releases

### ISSUE THREAD

**Issue by frnktrgr** [simplesamlphp/simplesamlphp#1533]
**Describe the bug**
The metarefresh module is not correctly installed under `modules` directory and therefore not working.
It is now located in `vendor/simplesamlphp/simplesamlphp-module-metarefresh`.

**Details**
In commit [4ee246c](https://github.com/simplesamlphp/simplesamlphp/commit/4ee246cb607e0dd6dcfdfded96121fc0a240d5dd#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R69) the required version of `simplesamlphp/simplesamlphp-module-metarefresh` was bumped from `^0.9` to `^1.0`. Version `1.0.1` requires `dev-master` of `simplesamlphp/simplesamlphp`; therefore version `1.0.0` got packaged in releases 1.19.2 and 1.19.3.

Version `1.0.0` has the following type in `composer.json`:
```json
    "type": "simplesamlphp-metarefresh",
```
instead of
```json
    "type": "simplesamlphp-module",
```
IMHO (_having no deeper knowledge about plugin installation ;-)_): this breaks the installation.

**Proposal**
Switch back to version `^0.9` of simplesamlphp-module-metarefresh. Verified with:
```
composer require simplesamlphp/simplesamlphp-module-metarefresh ^0.9
php composer.phar install
```

**Comment by iamCristYe** [comment]
possible to release a 1.19.4 to fix this?

**Comment by drjayvee** [comment]
Can this be released please?

I upgraded from 1.19.1 to solve the `Warning: session_create_id(): Failed to create new ID` warnings, only to find out that this issue made things even worse (users couldn't log in at all because the metadata wasn't getting refreshed).

I'd argue that either 1.19.4 should be released ASAP, or that 1.19.2 and 1.19.3 be *unpublished*.

**Comment by tvdijen** [comment]
Should be fixed in 1.19.4

**Comment by drjayvee** [comment]
I just did a quick test, and metarefresh works in 1.19.4.
